### PR TITLE
Routes: Skip domain

### PR DESCRIPTION
**Pull request by d00rman**
The URI layout has included the domain. However, the domain is not relevant for the producer side, so do not include it. The new URI now is `http://domain:port/v1/topics/{name}`

### CODE REVIEW

**Comment by d00rman** [comment]
cc @wikimedia/services

File: routes/v1.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ module.exports = function(appObj) {
 
     return {
         path: '/',
-        api_version: 1,
```
Comment (eevans) [inline routes/v1.js:124]: domain i understand, but why remove api_version?  Are we not versioning the API?

Comment (d00rman) [inline routes/v1.js:124]: Yeah, the trick is that currently you can either keep both the domain and the version or skip them both. Should probably change that though.

Comment (eevans) [inline routes/v1.js:124]: Clearly, the precedent is to use an HTTP resource for the version, so I'm sure I'm wading into a mess by asking, but what about using the `Accept` header?  i.e. `Accept: application/vnd.surge.v1+json`

Comment (gwicke) [inline routes/v1.js:124]: I'd be in favor of keeping the path-based versioning. Headers tend to be less discoverable, harder to use, and makes it hard to actually change the URL layout (which is the main reason for incrementing the major version in the path).

Comment (gwicke) [inline routes/v1.js:124]: @d00rman, could we cheat & set path to /v1/ ?

Comment (d00rman) [inline routes/v1.js:124]: Ha! Good idea @gwicke :)

**Comment by d00rman** [comment]
Done, the layout is now `/v1/topics/{name}`

**Pull request merged**